threeML/plugins/FermiLATLLELike.py

```diff
@@ -0,0 +1,487 @@
+        Return the name of the instrument and detector
+
+        :return:
+        """
+
+        return self._telescope
+
+    @property
+    def livetime(self):
+        return self._livetime
+
+    @property
+    def livetime_start(self):
+        return self._tstart
+
+    @property
+    def livetime_stop(self):
+        return self._tstop
+
+    def _compute_mission_times(self):
+
+        mission_dict = {}
+
+        if self.triggertime == 0:
+            return None
+
+        # Complements to Volodymyr Savchenko
+
+        xtime_url = "https://heasarc.gsfc.nasa.gov/cgi-bin/Tools/xTime/xTime.pl"
+
+        pattern = """<tr>.*?<th scope=row><label for="(.*?)">(.*?)</label></th>.*?<td align=center>.*?</td>.*?<td>(.*?)</td>.*?</tr>"""
+
+        args = dict(
+                time_in_sf=self.triggertime,
+                timesys_in="u",
+                timesys_out="u",
+                apply_clock_offset="yes")
+
+        if has_requests:
+
+            try:
+
+                content = requests.get(xtime_url, params=args).content
+
+                mission_info = re.findall(pattern, content, re.S)
+
+                mission_dict['UTC'] = mission_info[0][-1]
+                mission_dict[mission_info[7][1]] = mission_info[7][2]  # LIGO
+                mission_dict[mission_info[8][1]] = mission_info[8][2]  # NUSTAR
+                mission_dict[mission_info[12][1]] = mission_info[12][2]  # RXTE
+                mission_dict[mission_info[16][1]] = mission_info[16][2]  # SUZAKU
+                mission_dict[mission_info[20][1]] = mission_info[20][2]  # SWIFT
+                mission_dict[mission_info[24][1]] = mission_info[24][2]  # CHANDRA
+
+            except:
+
+                warnings.warn("You do not have the requests library, cannot get time system from Heasarc "
+                              "at this point.")
+
+                return None
+
+        else:
+
+            warnings.warn("You do not have the requests library, cannot get time system from Heasarc at this point.")
+
+            return None
+
+        return mission_dict
+
+    def peek(self):
+        """
+        Examine the currently selected interval
+        If connected to the internet, will also look up info for other instruments to compare with
+        Fermi.
+
+        :return: none
+        """
+
+        mission_dict = self._compute_mission_times()
+
+        fermi_dict = {}
+
+        fermi_dict['Fermi Trigger Time'] = self.triggertime
+        fermi_dict['Fermi MET OBS Start'] = self._start_events
+        fermi_dict['Fermi MET OBS Stop'] = self._stop_events
+        fermi_dict['Fermi UTC OBS Start'] = self._utc_start
+        fermi_dict['Fermi UTC OBS Stop'] = self._utc_stop
+
+        if mission_dict is not None:
+            mission_df = pd.Series(mission_dict)
+
+            display(mission_df)
+
+        fermi_df = pd.Series(fermi_dict)
+
+        display(fermi_df)
+
+
+def lle_light_curve_plot(time_bins, cnts, bkg, width, selection, bkg_selections):
```

Comment:
At some point we will need to tackle this plotting thing, because having plotting routines all over the place is going to get messy. Maybe we should collect them all in one place. That way we can also reduce a lot the duplication of code.